### PR TITLE
Fixed first message disappearing bug.

### PR DESCRIPTION
**Pull request by juliangarritano**
Added option to force user update. Forcing user update on appboot to ensure conversationStarted is correctly set on first message send.

### CODE REVIEW

**Comment by juliangarritano** [comment]
Note that this bug was only reproducible if you initialized SupportKit with ONLY the appToken.

The reason for this is that updateUser was not saving the user on init() because the model wasn't considered to be 'dirty' (none of the main properties had changed)

I fixed this by allowing a forced updateUser which is now called on init();

File: src/js/main.js [hunk 1]
```diff
@@ -216,12 +220,8 @@ var SupportKit = Marionette.Object.extend({
             return $.Deferred().resolve(this.user);
         }
 
-        this.user.set(userInfo, {
-            parse: true
-        });
-
         this._throttledUpdate = this._throttledUpdate || _.throttle(this._updateUser.bind(this), 60000);
-        return this._throttledUpdate();
+        return this._throttledUpdate(userInfo);
```
Comment (lemieux) [inline src/js/main.js]: hmm I think we'll lose data if we throttle ... what happens if we do 3 update user in a row? only the last set of attributes if taken in account?

Comment (juliangarritano) [inline src/js/main.js]: What do you suggest?

Comment (juliangarritano) [inline src/js/main.js]: I guess the set being in updateUser instead sort of fixed that problem...

Comment (lemieux) [inline src/js/main.js]: yeah, so what about leaving the set in updateUser, but do a set in the other place too and call _updateUser anyway?

Comment (juliangarritano) [inline src/js/main.js]: That would work :+1:

**Comment by lemieux** [comment]
@juliangarritano can you review my changes? I removed _updateUser as we discussed, but it lead to fixing the tests around that.

**Comment by juliangarritano** [comment]
Remove the call to updateUser in the chatController.

Rest is good :+1:

**Comment by jpjoyal** [comment]
Was this shipped ?

**Comment by lemieux** [comment]
No but it's in integration. Want to ship it now?

**Pull request merged**